gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Jan 21 18:56:58 IST 2019
+#Thu May 16 13:42:01 IST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-all.zip
```

Comment:
Change to 5.4.1

Reply:
Why?

Reply:
Because it's the latest version

Reply:
Please go through Travis logs - https://travis-ci.org/fossasia/open-event-android/builds/533186940?utm_source=github_status&utm_medium=notification

Reply:
`Minimum supported Gradle version is 5.1.1`

The minimum supported version is 5.1.1. Versions after that can also be used. So shouldn't we use the latest available version?